lib/activerecord-multi-tenant/query_monitor.rb

```diff
@@ -6,11 +6,34 @@ def self.enable_query_monitor; @@enable_query_monitor = true; end
   def self.query_monitor_enabled?; @@enable_query_monitor; end
 
   class QueryMonitor
-    def start(name, id, payload); end
-    def finish(name, id, payload)
+    def finish(_name, _id, _payload); end
```

Comment:
Could we instead leave the existing `finish` hook alone, and add our own code only to the `start` hook? I feel like this might make for a clearer diff from the upstream gem &mdash; and presumably leaving the existing `finish` hook in place won't cause us any issues? Just a thought :man_shrugging: 

The hook code itself looks good.